mqtt2influxdb/cli.py

```diff
@@ -17,6 +17,7 @@ def main():
     argp.add_argument('-c', '--config', help='path to configuration file (YAML format)', required=True)
     argp.add_argument('-D', '--debug', help='print debug messages', action='store_true')
     argp.add_argument('-t', '--test', help='test parse config', action='store_true')
+    argp.add_argument('-d', '--daemon', help='on connection error instead of exiting just wait for some time and try again', action='store_true')
```

Comment:
Maybe just leave --daemon, -d is used in another bc tools as shortcut for --device.